react/src/components/molecules/AccordionItem/index.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SvgCircleChevron from '../../atoms/icons/SvgCircleChevron';
+import SvgChevron from '../../atoms/icons/SvgChevron';
+import Heading from '../../atoms/headings/Heading';
+import Collapse from '../../animations/Collapse';
+import './style.css';
```

Comment:
I think this can change to the new method for including component styles now that #151 is merged

Reply:
not in here but i did update this in the style.scss file for this component